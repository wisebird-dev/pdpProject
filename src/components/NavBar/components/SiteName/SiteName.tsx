import React from 'react';
import Link from 'next/link';

import { Typography } from '@components';
import { SITE_NAME, landingPage } from '@core';

import useStyles from './SiteName.styles';

const SiteName: React.FC = () => {
    const classes = useStyles();

    return (
        <>
            <Link href={landingPage()}>
                <Typography
                    component="h1"
                    variant="h5"
                    color="textSecondary"
                    className={classes.siteNameMobile}
                >
                    {SITE_NAME.charAt(0)}
                </Typography>
            </Link>
            <Link href={landingPage()}>
                <Typography
                    component="h1"
                    variant="h5"
                    color="textSecondary"
                    className={classes.siteName}
                >
                    {SITE_NAME}
                </Typography>
            </Link>
        </>
    );
};

export default SiteName;
