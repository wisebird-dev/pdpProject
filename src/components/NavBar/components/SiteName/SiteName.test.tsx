import React from 'react';

import { render } from '@testUtils';

import SiteName from './SiteName';

describe('Sidemenu component', () => {
    it('matches snapshot', () => {
        const { container } = render(<SiteName />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
