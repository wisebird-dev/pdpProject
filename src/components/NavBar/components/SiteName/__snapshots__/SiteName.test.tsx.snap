// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidemenu component matches snapshot 1`] = `
<h1
  class="MuiTypography-root makeStyles-siteNameMobile-2 MuiTypography-h5 MuiTypography-colorTextSecondary"
>
  P
</h1>
`;
